### PR TITLE
docstring cleanup

### PR DESCRIPTION
**Pull request by robdaemon**
Adding the missing docstrings to make the linter complain a little less.

### CODE REVIEW

File: src/metabase/api/common/internal.clj [hunk 1]
```diff
@@ -55,16 +55,16 @@
...
-  [schema]
+  [schema route-str]
   (if-not schema
     ""
     (or (su/api-error-message schema)
         ;; Don't try to i18n this stuff! It's developer-facing only.
         (when config/is-dev?
           (log/warn
-           (u/format-color 'red (str "We don't have a nice error message for schema: %s\n"
+           (u/format-color 'red (str "We don't have a nice error message for schema: %s defined at %s\n"
```
Comment (robdaemon) [inline src/metabase/api/common/internal.clj:65]: I added this because it was hard to track down which endpoint it was complaining about.

Comment (camsaul) [reply src/metabase/api/common/internal.clj:65]: Good idea!

**Review by camsaul — APPROVED** [review]
Cool, thanks for doing this

**Comment by flamber** [comment]
Fixes #12417

**Pull request merged**